tests/ensure_provider_tests.py

```diff
@@ -57,8 +57,12 @@
 MINIMUM_VERSION = "minimum_version"
 EXTRA_RUN_CONDITION = "extra_run_condition"
 NM_ONLY_TESTS = {
-    "playbooks/tests_802_1x_updated.yml": {},
-    "playbooks/tests_802_1x.yml": {},
+    "playbooks/tests_802_1x_updated.yml": {
+        EXTRA_RUN_CONDITION: "ansible_distribution != 'RedHat' or\n      ansible_distribution_major_version | int < 9",
+    },
+    "playbooks/tests_802_1x.yml": {
+        EXTRA_RUN_CONDITION: "ansible_distribution != 'RedHat' or\n      ansible_distribution_major_version | int < 9",
+    },
```

Comment:
Maybe it is better to wrap the the line at 88 characters, see 
https://github.com/linux-system-roles/network/pull/405/commits/5b5a7d73724bef16d97631bb785176ff5bd97b71#diff-ad4940be7c7c31554d78a7255dab1bb445177228ef8acedefc688c04ed461975R124

other than that, LGTM.

Reply:
ok - how about this - https://github.com/linux-system-roles/network/commit/6c3bfc296d74272da9e0c58de35f2039f3e3e4f2